src/app.js

```diff
@@ -5,10 +5,14 @@ const path = require("path");
 const session = require("express-session");
 const AWS = require("aws-sdk");
 const DynamoDBStore = require("connect-dynamodb")(session);
-const featureSets = require("./app/fraud/featureSets");
```

Comment:
We will still need this while performing third party updates

Reply:
This line was just moved to line 14, so the 3 requires are consistent with dl/passport.